docs/en/serverless/quickstarts/k8s-logs-metrics.mdx

```diff
@@ -0,0 +1,45 @@
+---
+slug: /serverless/observability/quickstarts/k8s-logs-metrics
+title: Kubernetes logs and metrics
+description: Learn how to monitor your cluster infrastructure running on Kubernetes.
+tags: [ 'serverless', 'observability', 'how-to' ]
+---
+
+<p><DocBadge template="technical preview" /></p>
+
+In this quickstart guide, you'll learn how to create the Kubernetes resources that are required to monitor your cluster infrastructure. 
+
+This new approach requires minimal configuration and provides you with an easy setup to monitor your cluster. You no longer need to download, install, or configure the Elastic Agent, everything happens automatically when you run the kubectl command.
+
+The kubectl command installs the Elastic Agent in your Kubernetes cluster, downloads all the Kubernetes resources needed to collect metrics from the cluster, and sends it to Elastic.
+
+## Before you begin
+
+This quickstart has the following requirements:
+
+- The **Admin** role or higher is required to onboard system logs and metrics. To learn more, refer to <DocLink slug="/serverless/general/assign-user-roles" />.
+- You have a running Kubernetes cluster.
+- You have installed Kubectl. 
```

Comment:
We could include the [link to kubectl tool](https://kubernetes.io/docs/reference/kubectl) just in case.